packages/now-cli/src/util/dev/server.ts

```diff
@@ -511,11 +518,11 @@ export default class DevServer {
     this.getNowConfigPromise = null;
   };
 
-  getNowConfig(): Promise<NowConfig> {
+  getNowConfig(client?: Client, project?: Project): Promise<NowConfig> {
```

Comment:
Why is `client` optional?

Also why is `project` optional? Seems like we already error when no project is linked in `vc dev`.

Reply:
So we don't pass them around each call to getNowConfig

Reply:
They could be moved to ProjectSettings, or something else other than via `start()`?